pymc3/step_methods/hmc/nuts.py

```diff
@@ -96,6 +103,9 @@ class NUTS(BaseHMC):
             "energy": np.float64,
             "max_energy_error": np.float64,
             "model_logp": np.float64,
+            "process_time_diff_ns": np.int64,
+            "perf_counter_diff_ns": np.int64,
+            "perf_counter_ns": np.int64,
```

Comment:
Will these always be integers? It doesn't look like it in the plot you shared

Reply:
The nanosecond versions are. But for py3.6 compat I'm switching to float.